genesyscloud/telephony_providers_edges_phonebasesettings/resource_genesyscloud_telephony_providers_edges_phonebasesettings.go

```diff
@@ -85,9 +85,8 @@ func updatePhoneBaseSettings(ctx context.Context, d *schema.ResourceData, meta i
 	}
 
 	sdkConfig := meta.(*gcloud.ProviderMeta).ClientConfig
-	edgesAPI := platformclientv2.NewTelephonyProvidersEdgeApiWithConfig(sdkConfig)
-
-	phoneBaseSettings, resp, getErr := edgesAPI.GetTelephonyProvidersEdgesPhonebasesetting(d.Id())
+	pp := getPhoneBaseProxy(sdkConfig)
```

Comment:
change pp to phoneProxy for better readability

Reply:
addressed